templates/shrine/manifest.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Shrine",
+  "short_name": "Shrine by Polymer",
+  "icons": [{
+      "src": "https://app-layout-assets.appspot.com/assets/shrine/launcher-icon-128x128.png",
+      "sizes": "128x128",
+      "type": "image/png"
+    }, {
+      "src": "https://app-layout-assets.appspot.com/assets/shrine/auncher-icon-192x192.png",
+      "sizes": "192x192",
+      "type": "image/png"
+    },{
+      "src": "https://app-layout-assets.appspot.com/assets/shrine/launcher-icon-384x384.png",
+      "sizes": "384x384",
+      "type": "image/png"
+    }],
+  "start_url": "",
```

Comment:
I think this needs to be "index.html" or launching from homescreen will not work